scripts/cli-create-cosmos-table-api.sh

```diff
@@ -0,0 +1,35 @@
+LOCATION='eastus'
+RESOURCE_GROUP_NAME='rg-msdocs-tables-sdk-demo'
+COSMOS_ACCOUNT_NAME='cosmos-msdocs-tables-sdk-demo-123'
```

Comment:
Do we want to add the same comment regarding the account name as we have in the ps1 script? It may also be worth to add a comment about the resource group name so that people following the sample know they can also change it